test/e2e/crud.go

```diff
@@ -39,12 +51,6 @@ var _ = framework.KubeDescribe("Federated types [Feature:Federation][Experimenta
 			It("should be created, read, updated and deleted successfully", func() {
 				fedframework.SkipUnlessFederated(f.ClientSet)
 
-				// Load clients only if not skipping to avoid doing
-				// unnecessary work.  Assume clients can be shared
-				// across tests.
-				if clusterClients == nil {
-					clusterClients = f.GetClusterClients()
```

Comment:
Consider reverting this change.  Removing the previous skip logic from the e2e tests is orthogonal to the purpose of this PR.